draft-ietf-tls-tls13.md

```diff
@@ -1567,8 +1567,8 @@ cipher_suites
   the server does not recognize, support, or wish to use, the server
   MUST ignore those cipher suites, and process the remaining ones as
   usual. Values are defined in {{cipher-suites}}. If the client is
-  attempting a PSK key establishment, it MUST only advertise the single
-  cipher suite associated with the PSK.
+  attempting a PSK key establishment, it MUST advertise at least one
+  cipher suite containing a Hash associated with the PSK.
```

Comment:
I think I would rather it not be mentioned. On a server, the text should be ignored. On a client, it actually should be ignored too. Either you keep all your sessions in-memory, in which case you necessarily satisfy this and send the cipher list you would have sent anyway. Or you have some persistence and might offer a session from a prior configuration, in which case you probably ought to avoid offering that session but if you did, you certainly would still send the cipher list you would have sent anyway and not pick an arbitrary hash-matching cipher.

Moreover, having clients vary their preferences based on a stateful thing (session resumption) is very dangerous and risks turning TLS 1.3 into a deployment hazard. Deploying a server change and then rolling it back should be as close to a no-op as possible.